standard/recommendations/core/REC_client_side_filters.adoc

```diff
@@ -0,0 +1,7 @@
+[[rec_core_client_side_filters]]
+[width="90%",cols="2,6a"]
+|===
+^|*Recommendation {counter:rec-id}* |*/rec/core/client_side_filters*
+^|A |A WCMP record's link object advertising an MQTT channel SHOULD provide a `+filters+` object to describe the properties that can be used in support of client side filtering, using JSON Schema.
```

Comment:
I think the additional "..., using JSON schema" might still be confusing for readers... 

What is using JSON schema? Is it the filters object? or the client-side filtering?